src/io/flutter/run/FlutterDebugProcess.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2016 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.run;
+
+import com.intellij.execution.ExecutionResult;
+import com.intellij.execution.configurations.RunProfileState;
+import com.intellij.execution.ui.RunnerLayoutUi;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.DefaultActionGroup;
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.ui.GuiUtils;
+import com.intellij.ui.content.Content;
+import com.intellij.xdebugger.XDebugSession;
+import com.jetbrains.lang.dart.ide.runner.ObservatoryConnector;
+import com.jetbrains.lang.dart.ide.runner.server.vmService.DartVmServiceDebugProcessZ;
+import com.jetbrains.lang.dart.util.DartUrlResolver;
+import io.flutter.run.daemon.RunMode;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+
+public class FlutterDebugProcess extends DartVmServiceDebugProcessZ {
```

Comment:
Is this class only used for the 'run' button or for both run and debug? Can you document this, and update the README.md for this directory?